pkg/secrethub/fakeclient/client.go

```diff
@@ -10,13 +10,15 @@ import "github.com/secrethub/secrethub-go/pkg/secrethub"
 type Client struct {
 	AccessRuleService *AccessRuleService
 	AccountService    *AccountService
+	CredentialService *CredentialService
 	DirService        *DirService
+	IDPLinkService    *IDPLinkService
+	MeService         *MeService
 	OrgService        *OrgService
 	RepoService       *RepoService
 	SecretService     *SecretService
 	ServiceService    *ServiceService
 	UserService       *UserService
-	secrethub.ClientInterface
```

Comment:
The reason we've put this here is to ensure that `fakeclient.Client` always implements `secrethub.ClientInterface` completely (also if additional methods are added to the interface).

It's okay if we leave it out here, but then let's add `var _ secrethub.ClientIterface = (*Client)(nil)` at the top of this file, so that we still ensure that `fakeclient.Client` implements `secrethub.ClientIterface`.